src/rprblender/ui/object.py

```diff
@@ -72,6 +72,9 @@ def draw(self, context):
 
         col = self.layout.column()
         col.enabled = rpr.subdivision
-        col.prop(rpr, 'subdivision_factor')
-        col.prop(rpr, 'subdivision_crease_weight')
-        col.prop(rpr, 'subdivision_boundary_type')
+        if context.scene.rpr.render_quality == 'FULL2':
+            col.prop(rpr, 'subdivision_factor', text="Level")
+        else:
+            col.prop(rpr, 'subdivision_factor')
+            col.prop(rpr, 'subdivision_crease_weight')
```

Comment:
Creasing is still valid in RPR2?

Reply:
Yes, it works with RPR2. Restored usage and displayed UI once again